.github/CODEOWNERS

```diff
@@ -152,7 +152,6 @@ packages/office-ui-fabric-react/src/components/Tooltip/ @microsoft/cxe-red @beho
 packages/office-ui-fabric-react/src/components/Theme/ @dzearing
 
 ## Experiments
-packages/office-ui-fabric-react/src/components/Pagination/ @caradong
```

Comment:
I don't know of a substitute for this component, so I just removed it from the file. However, if anyone know of a good substitute let me know.